.circleci/config.yml

```diff
@@ -56,7 +61,8 @@ jobs:
     executor: node10
     steps:
       - checkout
-      - set_up_node_modules
+      - set_up_node_modules:
+          node_version: 10
```

Comment:
Where is node_version 12 used (or not added yet?) I see the default is 12 for `set_up_node_modules` but we are setting it to 10 down here.

Reply:
It's used in the 2 other test jobs. `test` and `test_full`.

Reply:
Ah, gotcha! I didn't see it here in the changes, but you can see the other sections when viewing the whole file

```yaml
jobs:
  test:
    executor: node12
    steps:
      - checkout
      - set_up_node_modules
      - run: yarn test --ci
  test_full:
    executor: node12
    steps:
      - checkout
      - set_up_node_modules
      - run: yarn test --full --ci
  test_full_10:
    executor: node10
    steps:
      - checkout
      - set_up_node_modules:
          node_version: 10
      - run: yarn test --full --ci
```